revolv/static/dashboard.scss

```diff
@@ -72,3 +91,116 @@
     max-width: 100%;
     float: left;
 }
+.project-container {
+    padding-left: 0px;
```

Comment:
For padding, you can specify all at once by using `padding: 0`. Also, in general you can specify padding in the form `padding: <top>, <right>, <bottom>, <left>`